worlds/bomb_rush_cyberfunk/docs/en_Bomb Rush Cyberfunk.md

```diff
@@ -0,0 +1,27 @@
+# Bomb Rush Cyberfunk
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and export 
+a config file.
+
+## What does randomization do in this game?
+
+The goal of Bomb Rush Cyberfunk randomizer is to defeat all rival crews in each borough of New Amsterdam, but several 
```

Comment:
[Optional] Passage could possibly be reordered to read a bit better? Something like
```markdown
## What does randomization do in this game?
The goal of Bomb Rush Cyberfunk randomizer is to defeat all rival crews in each borough of New Amsterdam.

Items can be found by picking up any type of collectible ...

## What changes have been made to the base game?
- The prelude in the police station can be skipped.
...
The mod also adds two new apps to the phone, ...
```

Reply:
Adjusted it